konrad/cloud.py

```diff
@@ -485,7 +487,9 @@ class HighCloud(DirectInputCloud):
     def update_cloud_profile(self, atmosphere, convection, **kwargs):
         """Keep the cloud attached to the convective top. """
         self.shift_cloud_profile(
-            norm_new=convection.get('convective_top_index')[0],
+            norm_new=np.int(
+                np.round(convection.get('convective_top_index')[0])
```

Comment:
You can omit the ``np.round`` as I hope that ``convective_top_index`` is always an integer. If that is not the case, that should be fixed instead 😉

Reply:
I actually changed this a while ago, so that the `convective_top_index` is not an integer, but corresponds to the `convective_top_temperature` and `convective_top_pressure`, which are between model levels, due to the interpolation. Then the `convective_top_index` is also between model levels, and not an integer. I'm thinking about changing it back though, so that it is the ''real'' convective top. It's not so obvious where that should be for the relaxed adjustment, but maybe just the closest level to the zero convective heating rate.

Reply:
Oh, I did not notice that 😇 Anyhow, I would support the idea to change it back to an ``int`` as I guess that is what you would expect for an ``*_index``. Of course, the temperature and pressure should be the interpolated values. IMHO the clarity does outweigh the small inconsistency between the methods. What do you think?